<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QTreeWidget" name="DataSettreeWidget">
      <column>
       <property name="text">
        <string>Header</string>
       </property>
      </column>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="TheoryLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="TheorytabWidget">
         <property name="tabShape">
          <enum>QTabWidget::Triangular</enum>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="tabsClosable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionNew_Theory">
   <property name="icon">
    <iconset resource="Theory.qrc">
     <normaloff>:/Icons/Images/Physics.ico</normaloff>:/Icons/Images/Physics.ico</iconset>
   </property>
   <property name="text">
    <string>New Theory</string>
   </property>
   <property name="toolTip">
    <string>Create Selected Theory</string>
   </property>
  </action>
  <action name="actionCalculate_Theory">
   <property name="icon">
    <iconset resource="Theory.qrc">
     <normaloff>:/Icons/Images/Bubble.ico</normaloff>:/Icons/Images/Bubble.ico</iconset>
   </property>
   <property name="text">
    <string>Calculate Theory</string>
   </property>
   <property name="toolTip">
    <string>Calculate Theroy</string>
   </property>
  </action>
  <action name="actionMinimize_Error">
   <property name="icon">
    <iconset resource="Theory.qrc">
     <normaloff>:/Icons/Images/Minimize.ico</normaloff>:/Icons/Images/Minimize.ico</iconset>
   </property>
   <property name="text">
    <string>Minimize Error</string>
   </property>
   <property name="toolTip">
    <string>Minimize Error</string>
   </property>
  </action>
  <action name="actionTheory_Options">
   <property name="icon">
    <iconset resource="Theory.qrc">
     <normaloff>:/Icons/Images/Settings.ico</normaloff>:/Icons/Images/Settings.ico</iconset>
   </property>
   <property name="text">
    <string>Theory Options</string>
   </property>
   <property name="toolTip">
    <string>Theory Options</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Symbols.qrc"/>
  <include location="Theory.qrc"/>
 </resources>
 <connections/>
</ui>
